fe/fe-core/src/main/java/com/starrocks/sql/ast/InsertStmt.java

```diff
@@ -107,6 +107,9 @@ public enum ColumnMatchPolicy {
     // create partition if not exists
     private boolean isDynamicOverwrite = false;
 
+    // if isFromOverwrite is true, it means that the insert statement is generated by the overwrite statement
+    private boolean isFromOverwrite = false;
```

Comment:
what's the different between `isOverwrite` and `isFromOverwrite`?

Reply:
check [InsertOverwriteJobRunner.java](https://github.com/StarRocks/starrocks/pull/55756/files#diff-9ba512541351f603a9bce0ba274658f639eafebe2fbf635d7ac26072f6ff4c81)
isOverwrite set to true at analyze phase, it will set to false when stmt execute.